CHANGE.md

```diff
@@ -0,0 +1,60 @@
+# Critical Updates for POD.jl
+
+I've forked the master branch hosted by lanl-ansi to my own Github while keeping all the most recent features merged on my master. Current, I am testing MPC instances with my master branch. Here is a summary of what are changed:
+
+## Merged Features
+* `disc_ratio` selection algorithm
+* logarithm embedding formulation
+* bounding model warm starting and no-good-cuts using solution pool
+* `disc_vars` selection algorithm based on weighted minimum vertex cover
+* support for binary/integer problem
+* support for `sin` and `cos` functions
```

Comment:
what is this support - I was under the impression we did not support sin and cosine

Reply:
This changelog is outdated. I will update the file content.